### ISSUE TITLE
Enemy download

### ISSUE THREAD

**Issue by Nickduino** [Sniffdog/Hashsynminer#15]
> Cannot find .\Bin\NVIDIA-Enemy2\z-enemy.exe distributed at https://mega.nz/#!5WACFRTT!tV1vUsFdBIDqCzBrcMoXVR2G9YHD6xqct5QB2nBiuzM.

I downloaded and unzipped it in \bin but you might want to check that out

**Comment by Sniffdog** [comment]
yeah I think it has everything to do with unpacking from a mega link....

For now you did the right process by adding to bin folder

I will keep working to get enemy to unpack properly... just wish the files where on github
